Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java

```diff
@@ -3,11 +3,9 @@
 import com.epam.test_generator.entities.Case;
 import com.epam.test_generator.services.CaseService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.*;
```

Comment:
It is considered bad practice to use wildcards in imports. Please import explicitly.